lib/services.js

```diff
@@ -250,16 +250,21 @@ const services = {
                     if (err) {
                         return cb(err, res);
                     }
-                    cb(null, res); // this is smart
```

Comment:
Was the idea here to allow the batch delete to happen in the background? Is it possible, then, with large batch delete operations the request will timeout?